src/box/box.cc

```diff
@@ -2540,6 +2540,12 @@ box_wait_quorum(uint32_t lead_id, int64_t target_lsn, int quorum,
 	return 0;
 }
 
+/**
```

Comment:
Please, extend the commit message, describe the bug and the solution. It's not convenient to go to the ticket, when you use `git log`

Reply:
Done.